src/features/Chat/components/ActiveWindow/Header/index.tsx

```diff
@@ -46,54 +71,63 @@ const Header = ({ chat }: Props) => {
   const isMentor = useAppSelector(selectIsMentor);
   const mentor = useAppSelector(selectMentorById(chat.buddyId));
 
-  const icons = {
-    ok: <ProfileIcon color="purpleDark" />,
-    archived: <img src={ArchivedIcon} />,
-    banned: <img src={BlockedIcon} />,
+  const [isReportModalOpen, setIsReportModalOpen] = useState(false);
+
+  const openDialog = async (variant: DialogVariant) => {
+    const isConfirmed = await getConfirmation({
+      borderColor: confirmDialogMap[variant].borderColor,
+      closeText: t('dialog.cancel'),
+      confirmId: `confirm-${variant}`,
+      confirmText: t(`dialog.${variant}.confirm`),
+      description: t(`dialog.${variant}.description`, {
+        buddyName: chat.displayName,
+      }),
+      title: t(`dialog.${variant}.title`),
+    });
+    if (isConfirmed) {
+      updateChatStatus({
+        userId,
+        buddyId: chat.buddyId,
+        status: confirmDialogMap[variant].targetFolder,
+      });
+    }
   };
```

Comment:
Looks great! Very intuitive.

The name could also be renamed from `openDialog` to `confirmAction` - then it would be more clear

Reply:
Changed!